homeassistant/components/fujitsu_hvac/config_flow.py

```diff
@@ -0,0 +1,145 @@
+"""Config flow for Fujitsu HVAC (based on Ayla IOT) integration."""
+from asyncio import timeout
+import logging
+from typing import Any
+
+from ayla_iot_unofficial import AylaAuthError, new_ayla_api
+from ayla_iot_unofficial.fujitsu_hvac import FujitsuHVAC
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import selector
+
+from .const import (
+    API_TIMEOUT,
+    AYLA_APP_ID,
+    AYLA_APP_SECRET,
+    CONF_DEVICE,
+    CONF_EUROPE,
+    DOMAIN,
+    NO_DEVICES_ERROR,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Required(CONF_EUROPE): bool,
+    }
+)
+
+
+async def get_devices(data: dict[str, Any]) -> dict[str, Any]:
```

Comment:
are you planning to reuse that method in an options for or something? Otherwise I'd just put it inline.